app/models/hyrax/permission_template.rb

```diff
@@ -79,11 +79,16 @@ def source
 
     ##
     # A bit of an analogue for a `belongs_to :source_model` as it crosses from Fedora to the DB
-    # @return [AdminSet, ::Collection]
-    # @raise [Hyrax::ObjectNotFoundError] when neither an AdminSet or Collection is found
+    # @return [AdminSet, ::Collection, Hyrax::AdministrativeSet, Hyrax::PcmdCollection]
+    # @raise [Hyrax::ObjectNotFoundError] when neither an admin set or collection is found
     def source_model
-      ActiveFedora::Base.find(source_id)
-    rescue ActiveFedora::ObjectNotFoundError
+      if Hyrax.config.disable_wings
+        source
```

Comment:
The `#source` method just above uses `query_service.find_by(:id)`.  So I didn't repeat that code here.  It seems likely that once fully Valkyrized, `#source_model` will be deprecated and replaced by `#source`.

Reply:
if this is to be deprecated (which seems right to me) why is it changing? would it be okay to switch to using `source` uniformly in `PermissionTemplateForm` instead?

Reply:
I can't see any reason not to deprecate `#source_model` now.  The 3rd commit does that.  The description in this PR identifies the uses of `#source_model` and how they were handled as part of the deprecation.